frontend/src/components/Banner.test.tsx

```diff
@@ -80,6 +104,29 @@ describe('Banner', () => {
     `);
   });
 
+  it('does not show troubleshooting link if warning', () => {
+    const tree = shallow(
+      <Banner message='Some message' mode='warning' showTroubleshootingGuideLink={true} />,
+    );
+    expect(tree).toMatchInlineSnapshot(`
```

Comment:
Same as above, please expect just the troubleshooting link doesn't show.
Warning mode snapshot was already tested above.